src/engines/html5/html5.js

```diff
@@ -1145,6 +1148,18 @@ export default class Html5 extends FakeEventTarget implements IEngine {
     }
   }
 
+  _handleWaiting(): void {
+    let playing = false;
+    this._eventManager.listenOnce(this._el, Html5EventType.PLAYING, () => (playing = true));
```

Comment:
Maybe [el.readyState](https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/readyState) can help  instead ?

Reply:
el.readyState is 2 once waiting in both real buffering and unmuting so it won't help